ui/src/components/mobileservices/BindingPanel.js

```diff
@@ -46,20 +90,65 @@ export class BindingPanel extends Component {
     this.open();
   }
 
-  componentWillMount() {
-    const serviceName = this.props.service.getName();
-    const schema = this.props.service.getBindingSchema();
-    const form = this.props.service.getFormDefinition();
-    const { service } = this.props;
+  isUPSService(){
+    return this.props.service.serviceClass &&
+        this.props.service.serviceClass.spec &&
+        this.props.service.serviceClass.spec.data &&
+        this.props.service.serviceClass.spec.data.externalName === "ag-unifiedpush-apb";
```

Comment:
https://lodash.com/docs/#get will make this much simpler

Reply:
will do.